routes/admin.route.js

```diff
@@ -56,4 +56,23 @@ router.put(
   adminController.putUpdateUser
 );
 
+router.post(
+  "/user",
```

Comment:
We can try to use a more descriptive route for now? Maybe '/disable' or '/disable-user'?